packages/fiori-mcp-server/src/tools/schemas/appgen-config-schema-props.ts

```diff
@@ -54,6 +52,19 @@ export const serviceOdata = z.object({
 });
 
 export const serviceCap = z.object({
+    servicePath: z
+        .string()
+        .describe('The odata endpoint. If the parameter is not provided, the agent should ask the user for it.')
+        .meta({
+            examples: [
+                '/odata/v4/<servicename>/',
+                '/odata/v4/MyRiskService/',
+                '/odata/v4/MyOdataV4Service/',
+                "/odata/v4/<relative '@path' annotation from service cds file>/",
+                "/<absolute '@path' annotation from service cds file>/",
+                '/myAbsolutePathFromServiceCdsFile/'
+            ]
+        }),
```

Comment:
```suggestion
servicePath: z
            .string()
            .describe(
                'The odata endpoint as provided by the cds mcp or as fallback in case that tool is not available from the service cds file.'
            )
            .meta({
                examples: [
                    '/odata/v4/<servicename>/',
                    '/odata/v4/MyRiskService/',
                    '/odata/v2/MyOdataV2Service/',
                    '/odata/v4/MyOdataV4Service/',
                    "/odata/v4/<relative '@path' annotation from service cds file>/",
                    "<absolute '@path' annotation from service cds file>/",
                    '/myAbsolutePathFromServiceCdsFile/'
                ]
            }),
```